src/Features/LanguageServer/ProtocolUnitTests/Ordering/RequestOrderingTests.cs

```diff
@@ -96,38 +96,6 @@ public async Task MutatingDoesntWaitForNonMutating()
             Assert.True(responses[2].StartTime < responses[1].EndTime);
         }
 
-        [Fact]
-        public async Task NonMutatingOperatesOnTheRightSolutions()
```

Comment:
This scenario is now covered by the DidChange_MultipleRequests test.

Reply:
Ah just saw this - I think we still may want some kind of coverage here to assert that the document's that are being acted on in the request are actually the correct versions.  DidChange_MultipleRequests only asserts the document tracker state, not that the document passed to the request handler is actually the correct version

Reply:
Yes, I'm planning to add more, but worth noting if the right document isn't passed to the second DidChange request in that test, the test fails.